lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -86,6 +140,8 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     accessibilityFeatures: computeAccessibilityFeatures(),
   );
 
+  ui.AppLifecycleState appLifecycleState = ui.AppLifecycleState.resumed;
```

Comment:
This global should belong in the instance of the `AppLifecycleStateSupport` class (it is in fact, the state that the class is supporting).